src/DssSpell.sol

```diff
@@ -59,187 +51,230 @@ contract DssSpellAction is DssAction {
+        // gap: 50 million USDS
+        // ttl: 24 hours
+        // tau: 7 days
+        // D3M Addresses:
+        // oracle: D3MOracle
+        // plan: D3MOperatorPlan
+        // pool: D3MAaveV3USDSNoSupplyCapTypePool
+        // aToken: TBC
+        // operator: 0x298b375f24CeDb45e936D7e21d6Eb05e344adFb5
+        // stabledebt address: TBC
+        // variabledebt address: TBC
+        // Additional Actions
+        // Expand DIRECT_MOM breaker to also include new D3M
+        // Note: this is already done within D3MInit.sol line 232
+
+        // D3MInstance memory d3m = D3MInstance({
+        //     plan:   DIRECT_SPK_AAVE_LIDO_USDS_PLAN,
+        //     pool:   DIRECT_SPK_AAVE_LIDO_USDS_POOL,
+        //     oracle: DIRECT_SPK_AAVE_LIDO_USDS_ORACLE
+        // });
+
+        // D3MCommonConfig memory d3mCfg = D3MCommonConfig({
+        //     hub:         D3M_HUB,
+        //     mom:         D3M_MOM,
+        //     ilk:         "DIRECT-SPK-AAVE-LIDO-USDS",
+        //     existingIlk: false,
+        //     maxLine:     100 * MILLION * RAD,
+        //     gap:         50 * MILLION * RAD,
+        //     ttl:         24 hours,
+        //     tau:         7 days
+        // });
+
+        // D3MAaveUSDSPoolConfig memory aaveCfg = D3MAaveUSDSPoolConfig({
+        //     king:         address(0), // TODO: replace with TBC
+        //     ausds:        address(0), // TODO: replace with TBC
+        //     usdsJoin:     address(0), // TODO: replace with TBC
+        //     usds:         address(0), // TODO: replace with TBC
+        //     stableDebt:   address(0), // TODO: replace with TBC
+        //     variableDebt: address(0) // TODO: replace with TBC
+        // });
+
+        // D3MOperatorPlanConfig memory operatorCfg = D3MOperatorPlanConfig({
+        //     operator: DIRECT_SPK_AAVE_LIDO_USDS_OPERATOR
+        // });
+
+        // D3MInit.initCommon({
+        //     dss:     dss,
+        //     d3m:     d3m,
+        //     cfg:     d3mCfg
+        // });
+
+        // D3MInit.initAaveUSDSPool({
+        //     dss:     dss,
+        //     d3m:     d3m,
+        //     cfg:     d3mCfg,
+        //     aaveCfg: aaveCfg
+        // });
+
+        // D3MInit.initOperatorPlan({
+        //     d3m: d3m,
+        //     operatorCfg: operatorCfg
+        // });
+
+        // ---------- Update WBTC Legacy Vaults Parameters  ----------
+        // Forum: https://forum.makerdao.com/t/wbtc-changes-and-risk-mitigation-10-august-2024/24844/48
+
+        // Decrease liquidation penalty for WBTC-A, WBTC-B, and WBTC-C from 13% to 0%
+        DssExecLib.setIlkLiquidationPenalty("WBTC-A", 0);
+        DssExecLib.setIlkLiquidationPenalty("WBTC-B", 0);
+        DssExecLib.setIlkLiquidationPenalty("WBTC-C", 0);
+
+        address spotter = DssExecLib.spotter();
+
+        // WBTC-A: Increase LERP for liquidation ratios from 145% to 150% over 6 days
+        DssExecLib.linearInterpolation({
+            _name:      "WBTC-A Offboarding",
```

Comment:
This considers as parameter update since offboarding is paused. So I think we can update the naming here.

Reply:
Sounds good to me

Reply:
fixed in https://github.com/makerdao/spells-mainnet/commit/b9f1c6a61f16a2006952aa42dd6fa8b2fb38a959